fe/fe-core/src/main/java/com/starrocks/connector/parser/trino/Trino2SRFunctionCallTransformer.java

```diff
@@ -148,6 +149,18 @@ private static void registerDateFunctionTransformer() {
                 ImmutableList.of(Expr.class));
     }
 
+    private static void registerStringFunctionTransformer() {
```

Comment:
please add some demo to the problem summary.

Reply:
Done